File: 프로그래밍/py/code/02.전체.py
name = input("이름을 입력하세요: ")
kor, eng = map(int, input("국어와 영어 점수를 입력하세요: ").split())

total = kor + eng
print(name, "님의 합계는", total, "점,", "평균은", total / 2, "점입니다", sep=" ")

File: 프로그래밍/c/code/02.전체.c
#include <stdio.h>

int main(void) {
    int kor, eng;

    printf("국어와 영어 점수를 입력하세요: ");
    scanf("%d %d", &kor, &eng);

    printf("합계 %d점, 평균 %.1f점\n", kor + eng, (kor + eng) / 2.0);
    return 0;
}
